### PR TITLE
13 bart formula

### LINKED ISSUE #13: new tap: bart / bartholomew
Installing `spin` via homebrew was a real joy. I'd like to have a tap for `bart`, too.

Were this request to be officially implemented, then documentation should be updated, too. Like the [Bartholomew quickstart page](https://developer.fermyon.com/bartholomew/quickstart) installation instructions.

And while I've not-yet made a homebrew tap, I am always willing to learn something new. So I am going to fork this repo and see what I can break on my computer. Yay! \o/ 

No promises, but I expect that I will give this a go later tonight and/or tomorrow. Cheers!

### PR DESCRIPTION
**Pull request by dimitrilw**
Should fix #13 . Also uses a test routine that validates installed version number.

### CODE REVIEW

**Comment by itowlson** [comment]
@dimitrilw Could you sign the commits for DCO purposes please?  You'll probably also need to GPG-sign them (I'm not sure about this repo but a lot of our repos require verified GPG-signed commits).  Thanks!

File: Formula/bart.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  end
+
+  if OS.linux? && Hardware::CPU.intel?
+    url "https://github.com/fermyon/bartholomew/releases/download/v0.9.0/bart-v0.9.0-linux-amd64.tar.gz"
+    sha256 "6a95803c3c62f67efccbaed1ebf506fdbe458f22e31fad66622b434265539d64"
+  end
+
+  if OS.linux? && Hardware::CPU.arm?
+    url "https://github.com/fermyon/bartholomew/releases/download/v0.9.0/bart-v0.9.0-linux-aarch64.tar.gz"
+    sha256 "5b031b864e03883ddb740aab8fd4ffaf306d043ba86f90bc5c123cef1ea7f006"  bart-v0.9.0-linux-aarch64.tar.gz
```
Comment (kate-goldenring) [inline Formula/bart.rb]: Looks like this was accidentally added in
```suggestion
    sha256 "5b031b864e03883ddb740aab8fd4ffaf306d043ba86f90bc5c123cef1ea7f006" 
```

**Review by kate-goldenring — COMMENTED** [review]
Thank you for adding this in! Just a quick fix and signoff on commits and should be good

File: Formula/bart.rb (hunk 1 above)
Comment (dimitrilw) [reply Formula/bart.rb]: Derp. That is just 100% "derp". :| Fixing.

**Comment by dimitrilw** [comment]
> @dimitrilw Could you sign the commits for DCO purposes please? You'll probably also need to GPG-sign them (I'm not sure about this repo but a lot of our repos require verified GPG-signed commits). Thanks!

I'll get that set-up today.

**Comment by dimitrilw** [comment]
Thank you for advising me to add GPG verification. Done.

image

File: Formula/bart.rb (hunk 1 above)
Comment (dimitrilw) [reply Formula/bart.rb]: Fixed.

**Comment by dimitrilw** [comment]
I see that the DCO req is for every merged commit. The "Details" advises me to rebase & resubmit. Doing that now.

image

I apologize for the Github-noob error. I have not used this feature of Github before (as you can tell) and I'm learning as I go.

**Comment by dimitrilw** [comment]
Done.

**Comment by dimitrilw** [comment]
I see the fails & I'm working to fix. Thank you for being patient with me -- first Homebrew Formula submission, so it is a learning experience.

**Comment by dimitrilw** [comment]
Updated.

**Comment by dimitrilw** [comment]
Also re-updated GPG signature on last commit -- I apologize. I did not realize that I needed to enable git signatures & set as default behavior in:
- git CLI
- VS-Code git
- the git extension I use in VS-Code

Adding that note here in case it helps anybody else.

**Comment by dimitrilw** [comment]
Gah! I apologize. My VS-Code settings are adding that last LF when I commit & push. Manually fixing outside VS-Code. ...I hope I have entertained you all with these errors. 😊

**Comment by dimitrilw** [comment]
Of note: I do not understand the other fail on MacOS-13. It shows:
image

**Comment by kate-goldenring** [comment]
Thanks for all the iterations here @dimitrilw! Do you mind signing your latest commit? My assumption is that the macos workflow there was canceled because the ubuntu workflow errored but i have not dug into it

**Comment by dimitrilw** [comment]
gah! I thought I had git config'd to auto-sign when I did `git commit` from CLI. ......so many potholes on this road. 😄

**Comment by dimitrilw** [comment]
Done. Thank you for being patient with me.

**Comment by itowlson** [comment]
Thank _you_ @dimitrilw for having the patience to keep at it despite all those potholes!

**Review by kate-goldenring — APPROVED** [review]
Thank you @dimitrilw for working through all the bot suggestions! I created an issue to figure out how to add windows in the future: https://github.com/fermyon/homebrew-tap/issues/16

**Comment by dimitrilw** [comment]
and just closing loop on this:
image

🥳

**Pull request merged**